### ISSUE TITLE
Small bug in example with Python 3 (including solution)

### ISSUE THREAD

**Issue by salihkilic** [enigmampc/catalyst#238]
Dear Catalyst Maintainers,

First of all, I'm not an experience programmer, I just found a solution to a simple bug in one of the examples.
I found a bug in the [dual_moving_average example](https://github.com/enigmampc/catalyst/blob/master/catalyst/examples/dual_moving_average.py). 

On line 88 it says:
`exchange = list(context.exchanges.values())[0]`

This only works in Python 2 because dict_values don't behave as a list anymore in Python 3. A stack overflow solution said it's more like a set or a view.

By changing line 88 to:
`base_currency = list(context.exchanges.values())[0].base_currency.upper()`

It should work in either version of Python.

**Comment by AvishaiW** [comment]
hi @salihkilic, Thank you for noting that out! I was looking through the code, trying to reproduce your error and noticed the code consists of the following two lines already: (lines 88-89)
`exchange = list(context.exchanges.values())[0]`
`base_currency = exchange.base_currency.upper()`
perhaps you might be using an older version of the code?
I must note out that some of the examples are old, and we are working on renewing them, so it's possible that they're a bit out of date.

**Comment by benibienz** [comment]
To add to this, the code example here is different from (and worse than) the one on the beginner tutorial page here: https://enigmampc.github.io/catalyst/beginner-tutorial.html

Should be an easy fix to update it. Although it's still in alpha, I'm sure many people new to catalyst would appreciate working introductory examples.

**Comment by lacabra** [comment]
Dear @salihkilic and @benibienz Thanks for reporting this issue. As @AvishaiW reports above, the code on that particular line was fixed on commit 93ebbf8b1f4fe22fa157903465d706a7b26a8f40 dated from Jan 9th, 2018, so @salihkilic might have been looking at an older version of the code. 

@benibienz reported a different issue that was fixed yesterday on commit 4337abd60a4bfa2f8335dba2c8c3bb9173f38cda that links the examples in the documentation to the actual source code so that they are both in sync.

In researching about these issues I uncovered a different issue that has also been fixed in 25e9f0f58f16651c86570c55ca704153d98dd35c that now makes the code run as documented at https://enigmampc.github.io/catalyst/beginner-tutorial.html#access-to-previous-prices-using-history, and I have validated that the example runs as expected.
